src/UnivPoly.jl

```diff
@@ -0,0 +1 @@
+denominator(f::UniversalPolyRingElem{QQFieldElem}) = denominator(data(f))
```

Comment:
Add a comment above that explains what this file is about?

Reply:
I tried to mimic the headers of some other files in `src`.